modules/ROOT/pages/release-notes.adoc

```diff
@@ -119,6 +119,28 @@ The property allowing to enable/disable xref:identity:rest-api-authorization.ado
 
 == Bug fixes
 
+=== Fixes in Bonita 2024.1-u3 (2024-07-18)
+
+==== Fixes in Bonita Runtime including Bonita Applications
+
+* RUNTIME-1883 - Add extra java 17 open packages instructions for Bundle packaging
+* RUNTIME-1697 - Avoid reading configuration from database, in favor to caching +
+Bonita Runtime makes heavy usage of *cache* to improve performance. In that context, most configuration changes require *server restart* to be taken into account. In particular, all SSO activations and configuration changes fall into this category (OIDC, SAML, Kerberos, JAAS, ...). +
+* RUNTIME-1877 - Session Sharing x sameSiteCookies value systematically set to "lax" when cluster is enabled. xref:runtime:install-a-bonita-bpm-cluster.adoc[New configuration properties] specific to cluster mode and session cookie have been added.
+* RUNTIME-1884 - `password` attribute silently removed from `org.bonitasoft.engine.identity.impl.UserImpl` class. Add removal information in the release note. Make the XStream deserialization less strict to avoid deserialization exception when a some fields are unknown.
```

Comment:
```suggestion
* RUNTIME-1884 - `password` attribute silently removed from `org.bonitasoft.engine.identity.impl.UserImpl` class. Add removal information in the release note. Make the XStream deserialization less strict to avoid deserialization exception when some fields are unknown.
```